src/master-playlist-controller.js

```diff
@@ -369,10 +327,20 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
-  useAudio(label) {
-    let audioEntry;
-    let newAudioPlaylistLoader;
-    let mediaGroupName = this.masterPlaylistLoader_.media().attributes.AUDIO;
+  useAudio() {
+    let media = this.masterPlaylistLoader_.media();
+    let master = this.masterPlaylistLoader_.master;
+
+    if (!media || !media.attributes || !media.attributes.AUDIO ||
+        !master.mediaGroups || !master.mediaGroups.AUDIO) {
+      return;
+    }
+    let mediaGroupName = media.attributes.AUDIO;
+
+    if (!master.mediaGroups.AUDIO[mediaGroupName]) {
+      return;
+    }
+    let audioEntries = master.mediaGroups.AUDIO[mediaGroupName];
```

Comment:
This is more for me, but do you know if we're doing all of our let declarations at the top of the function, except for variables declared within inner blocks, or are we doing the declaration when we define it within the function?